lib/matplotlib/artist.py

```diff
@@ -1,8 +1,7 @@
 from collections import namedtuple
 import contextlib
 from functools import wraps
-import inspect
-from inspect import Signature, Parameter
+from inspect import getdoc, Parameter, signature, Signature
```

Comment:
let's stick to (usually) not importing functions and referring to them by qualified name (https://peps.python.org/pep-0008/#imports states that "when importing a class [...], it is usually okay to spell this [as a from...import]", which implicitly suggests that a keeping the qualified name is usually preferred for non-classes).